sevntu-checks/sevntu-checks.xml

```diff
@@ -80,6 +80,10 @@
       <property name="forbiddenClassesRegexpProtected" value=""/>
       <property name="forbiddenClassesRegexpPublic" value=""/>
     </module>
+    <module name="com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodsCheck">
+      <property name="methodName" value="exit"/>
```

Comment:
We need to rename Check to be ForbidCertainMethodCheck , as property is already "method"

Reply:
Done.